frontend/src/components/VariantListPage/VariantsTable.tsx

```diff
@@ -681,6 +681,20 @@ const VariantsTable: FC<VariantsTableProps> = ({
     notIncludedVariants && notIncludedVariants.has(variant.id) ? 1 : 0
   );
 
+  const listRef = useRef<FixedSizeList | null>(null);
+
+  useEffect(() => {
+    if (searchText) {
+      const matchIndex = sortedVariants.findIndex((variant) =>
+        variant.id.toLowerCase().includes(searchText.toLowerCase())
+      );
+
+      if (matchIndex !== -1 && listRef.current) {
+        listRef.current.scrollToItem(matchIndex, "start");
+      }
```

Comment:
**suggestion**: Here we could use `min` to add some padding to also provide a few rows of context for variants above this one. 

Something like:

```
if (matchIndex !== -1 && listRef.current) {
    const topPadding = Math.min(3, matchIndex);
    listRef.current.scrollToItem(matchIndex - topPadding, "start");
}
```

Would add at most 3 rows of padding above our first matched variant, which may be useful for even more context.

e.g.
![Screenshot 2024-12-12 at 15 09 15](https://github.com/user-attachments/assets/b2179d9d-3c02-4ac8-b430-e8cccc6d50c1)